recipe/meta.yaml

```diff
@@ -68,6 +68,7 @@ requirements:
   - curl
   - eigen
   - embree
+  - ffmpeg>=3.4.1
```

Comment:
Do we need to pin in the env file too?

Reply:
only seems to be on issue on build on linux? I can make the add anyways.

Reply:
Okay, seems fine to just have it here then.